src/app/shell/personal-cabinet/provider/create-workshop/create-description-form/create-description-form.component.html

```diff
@@ -3,12 +3,16 @@
   <app-image-form-control [imgMaxAmount]="10" [label]="'Фото'" formControlName="image">
   </app-image-form-control>
 
-  <label class="step-label"> Опис <span class="step-required">*</span></label>
+  <div class="step-label-combined">
+    <label class="step-label" > Опис <span class="step-required">*</span></label>
+    <label class="step-letter-count" fxLayout='row' fxLayoutAlign='end'> {{letterCount.length}}/<span>{{500 -
```

Comment:
it does not follow requirements, you should display characters amount / max length